geomapshark/static/css/gpf.css

```diff
@@ -12,6 +12,16 @@
   border: 2px solid rgba(252, 200, 70, 1);
 }
 
+#homepage_main_apps {
```

Comment:
Why is this duplicated? If this file is copied here by `collectstatic`, it shouldn't be versioned.